src/components/expression/expression-loader.js

```diff
@@ -170,6 +169,22 @@ genotet.ExpressionLoader.prototype.loadTfaProfile_ = function(fileName,
 };
 
 /**
+ * Loads expression data list into genotet.data.expressionFiles.
+ */
+genotet.ExpressionLoader.prototype.loadExpressionList = function() {
+  var params = {
+    type: 'list-expression'
+  };
+  this.get(genotet.data.serverURL, params, function(data) {
+    genotet.data.bindingFiles = [];
```

Comment:
Agreed. But how to send signal to dialogs after loading expression list in genetet.data.loadExpressionList?